espnet/nets/pytorch_backend/encoders.py

```diff
@@ -24,19 +24,23 @@ class BRNNP(torch.nn.Module):
     :param str typ: The RNN type
     """
 
-    def __init__(self, idim, elayers, cdim, hdim, subsample, dropout, typ="lstm"):
+    def __init__(self, idim, elayers, cdim, hdim, subsample, dropout, typ="blstm"):
         super(BRNNP, self).__init__()
+        bidir = typ[0]=="b"
         for i in six.moves.range(elayers):
             if i == 0:
                 inputdim = idim
             else:
                 inputdim = hdim
-            rnn = torch.nn.LSTM(inputdim, cdim, dropout=dropout, num_layers=1, bidirectional=True,
-                                batch_first=True) if typ == "lstm" \
-                else torch.nn.GRU(inputdim, cdim, dropout=dropout, num_layers=1, bidirectional=True, batch_first=True)
-            setattr(self, "birnn%d" % i, rnn)
+            rnn = torch.nn.LSTM(inputdim, cdim, dropout=dropout, num_layers=1, bidirectional=bidir,
+                                batch_first=True) if "lstm" in typ \
+                else torch.nn.GRU(inputdim, cdim, dropout=dropout, num_layers=1, bidirectional=bidir, batch_first=True)
+            setattr(self, typ + "%d" % i, rnn)
```

Comment:
@sw005320 Do you want to keep model compatibility?
If so, we should use `birnn` or `rnn` for attibutes.

Reply:
yes, let's do it. @JaejinCho, change it accordingly.